v22.1/copy-from.md

```diff
@@ -185,11 +217,68 @@ SELECT * FROM setecastronomy;
 ~~~
             name            |              phrase
 ----------------------------+------------------------------------
-  My name is Werner Brandes | "My" "voice" "is" "my" "passport"
   My name is Werner Brandes | My voice is my passport
+  My name is Werner Brandes | "My" "voice" "is" "my" "passport"
 (2 rows)
 ~~~
 
+#### Copy CSV delimited data from stdin with hex encoded byte array data
+
+To copy CSV data into CockroachDB and specify that CockroachDB should ingest hex encoded byte array data, enter the following statements:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE TABLE mybytes(a INT PRIMARY KEY, b BYTEA);
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS mybytes(a INT PRIMARY KEY, b BYTEA);
```

Reply:
Updated to that, thanks! Always a good practice